recipes/alchemiscale/meta.yaml

```diff
@@ -0,0 +1,102 @@
+{% set name = "alchemiscale" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: alchemiscale
+  version: {{ version }}
+
+source:
+  url: https://github.com/openforcefield/alchemiscale/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: d46f372adcaa257eb909587f668e791915188795b60db0c11bbfd6bfe9ac1a84
+
+build:
+  number: 0
+  skip: true
+
+outputs:
+  - name: alchemiscale-compute
+    script: build_package.sh
+    build:
+      noarch: python
+
+    requirements:
+      host:
+        - python >=3.9
+        - pip
+      run:
+        - python >=3.9
+        - gufe=0.9.1
+        - openfe=0.11.0
+        - requests
+        - click
+        - httpx
+        - pydantic<2.0
+        - openmoltools
+        - cloudpathlib
+        - dask
+        - distributed
+        - numba
+        - pymbar >=3.0.6,<4
+        - async-lru
+    
+    test:
+      imports:
+        - alchemiscale
+      requires:
+        - pip
+      commands:
+        - pip check
+
+  - name: alchemiscale-client
+    script: build_package.sh
+    build:
+      noarch: python
+
+    requirements:
+      host:
+        - python >=3.9
+        - pip
+      run:
+        - python >=3.9
+        - gufe=0.9.1
+        - openfe=0.11.0
+        - requests
+        - click
+        - httpx
+        - pydantic<2.0
+        - rich
+        - openmoltools
+        - cloudpathlib
+        - dask
+        - distributed
+        - numba
+        - pymbar >=3.0.6,<4
+        - async-lru
+        - nest-asyncio
```

Comment:
Is this package alchemiscale-compute with this extra dependency? Is so you should avoid a dual build and depend on alchemiscale-compute here instead.

Reply:
See https://github.com/conda-forge/matplotlib-feedstock/blob/main/recipe/meta.yaml for an example.

Reply:
In a perfect world, I would have some sort of alchecmiscale-base or have one depend on other other as you propose, but we are still moving very quickly on this development wise and in an iteration or two, the `compute` and `client` will be much more independent. So I figured once the dust settles on that I would re-work the dependencies. 

Thank you for the suggestion!

Reply:
I'm a bit busy at the moment with some personal things to resolve. Can you request a new reviewer. Thanks!

Reply:
Yea I don't get the extra overhead right now if this is a single source with different deps 🤔 Won't it be easier to address that in the feedstock once there and make the review simpler / quicker?